Apps/W1/External File Storage - Azure File Service Connector/app/src/ExtFileShareAccount.Page.al

```diff
@@ -0,0 +1,68 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.ExternalFileStorage;
+
+/// <summary>
+/// Displays an account that was registered via the File Share connector.
+/// </summary>
+page 4570 "Ext. File Share Account"
+{
+    SourceTable = "Ext. File Share Account";
+    Caption = 'Azure File Share Account';
+    Permissions = tabledata "Ext. File Share Account" = rimd;
+    PageType = Card;
+    Extensible = false;
+    InsertAllowed = false;
+    DataCaptionExpression = Rec.Name;
+    ApplicationArea = All;
+    UsageCategory = None;
+
+    layout
+    {
+        area(Content)
+        {
+            field(NameField; Rec.Name)
+            {
+                ShowMandatory = true;
+                NotBlank = true;
+            }
+            field(StorageAccountNameField; Rec."Storage Account Name") { }
+            field("Authorization Type"; Rec."Authorization Type") { }
+            field(SecretField; Secret)
+            {
+                Caption = 'Password';
+                Editable = SecretEditable;
+                ExtendedDatatype = Masked;
+                ToolTip = 'Specifies the Shared access signature Token or SharedKey.';
+
+                trigger OnValidate()
+                begin
+                    Rec.SetSecret(Secret);
+                end;
+            }
+            field(FileShareNameField; Rec."File Share Name") { }
+            field(DisabledField; Rec.Disabled) { }
+        }
+    }
+
+    var
+        SecretEditable: Boolean;
+        [NonDebuggable]
+        Secret: Text;
+
+    trigger OnOpenPage()
+    begin
+        Rec.SetCurrentKey(Name);
+
+        if not IsNullGuid(Rec."Secret Key") then
```

Comment:
Shouldn't the trigger `trigger OnAfterGetRecord()` be used for checks on the record?

I would suggest to move the code
```al
if not IsNullGuid(Rec."Secret Key") then
    Secret := '***';
```
to the `OnAfterGetRecord` trigger.
(This applies for the other external file storages, too.)